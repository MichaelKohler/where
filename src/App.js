import React from 'react';
import Overview from './Overview';
import Footer from './Footer';
import './scss/app.scss';

const App = () => {
  return (
    <div>
      <Overview />
      <Footer />
    </div>
  );
};

export default App;
