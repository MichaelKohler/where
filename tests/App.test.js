import React from 'react';
import { shallow } from 'enzyme';

import App from '../src/App';

test('should correctly shallow render App', () => {
  const component = shallow(<App />);
  expect(component).toMatchSnapshot();
});
