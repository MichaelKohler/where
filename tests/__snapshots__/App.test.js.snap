// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly shallow render App 1`] = `
<div>
  <Overview />
  <Footer />
</div>
`;
